sim/priest/glyphs.go

```diff
@@ -0,0 +1,57 @@
+package priest
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/core/proto"
+)
+
+func (priest *Priest) ApplyGlyphs() {
+	// Glphys
+	if priest.HasGlyph(int32(proto.PriestPrimeGlyph_GlyphOfShadowWordPain)) {
```

Comment:
btw I didnt notice the first time that you used the `HasGlyph` func. There should be explicit functions for checking `HasPrimeGlyph`, `HasMajorGlyph` etc.